### PR TITLE
[R2]: Document the endpoint location on the tokens page

### PR DESCRIPTION
**Pull request by vlovich**
Doesn't appear to actually be referenced anywhere.

### CODE REVIEW

**Comment by KianNH** [comment]
Might be worth changing the `<account>` to `<accountid>` and maybe link to https://developers.cloudflare.com/fundamentals/get-started/basic-tasks/find-account-and-zone-ids/ somewhere for people that aren't aware of what/where to find those - otherwise it's hidden in a specific bucket's settings tab.

**Comment by Erisa** [comment]
+1 to @KianNH's suggestion, looks good otherwise!

File: content/r2/platform/s3-compatibility/api.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ pcx-content-type: reference
 # S3 API compatibility
 
 R2 implements the S3 API to allow users and their applications to migrate easily. When comparing to AWS S3, Cloudflare has removed some API operations' features and added others. The S3 API operations are listed below with their current implementation status. Feature implementation is currently in progress. Refer back to this page for updates.
+The API is available via the `https://<account>.r2.cloudflarestorage.com` endpoint.
```
Comment (deadlypants1973) [inline content/r2/platform/s3-compatibility/api.md]:
```suggestion
The API is available via the `https://<ACCOUNT_ID>.r2.cloudflarestorage.com` endpoint. Find your [account ID in the Cloudflare dashboard](https://developers.cloudflare.com/fundamentals/get-started/basic-tasks/find-account-and-zone-ids/).
```

File: content/r2/platform/s3-compatibility/tokens.md [hunk 2]
```diff
@@ -26,4 +26,4 @@ You will not be able to access your **Secret Access Key** again after this step.
 
 {{</Aside>}}
 
-
+The S3 endpoint is available via `https://<account>.r2.cloudflarestorage.com`.
```
Comment (deadlypants1973) [inline content/r2/platform/s3-compatibility/tokens.md]:
```suggestion
The S3 endpoint is available via `https://<ACCOUNT_ID>.r2.cloudflarestorage.com` endpoint. Find your [account ID in the Cloudflare dashboard](https://developers.cloudflare.com/fundamentals/get-started/basic-tasks/find-account-and-zone-ids/).

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**